### ISSUE TITLE
Z-sorting issues on adjacent cells

### ISSUE THREAD

**Issue by reaperrr** [OpenRA/OpenRA#13430]
It seems that with DepthBuffer enabled, there are some issues where animations which should have translucency actually completely 'drown' pixels of graphics on adjacent cells:
![alpha-issue](https://cloud.githubusercontent.com/assets/2857877/26747969/f5452d40-47f9-11e7-9aa0-95d4f81ef870.png)

**Comment by pchote** [comment]
Indeed.  The WorldRenderer ordering needs to be redone for depth buffer enabled mods.  This is the same issue that causes problems with shadows.

**Comment by GraionDilach** [comment]
Is my screenshot showcasing the same issue?

![image](https://user-images.githubusercontent.com/1136302/71314488-9341d880-2449-11ea-9020-0dd53860ee62.png)

**Comment by pchote** [comment]
No. Voxels aren't composited into the scene with depth at all - which is a different issue.
